src/tests/unit/sync/syncimages_ring.f90

```diff
@@ -0,0 +1,25 @@
+! SYNC IMAGES([this_image - 1, this_image + 1]) with the STAT=STAT_STOPPED_IMAGE
+! specifier and wrap around of image numbers.  The test is intended to check
+! that syncing in a ring with a stopped image still terminates all images.
```

Comment:
@vehre: Great test idea! Can you please rename the test to better reflect its purpose? Right now it sounds as though it is meant to test if a ring correctly syncs. A better name might be something like `abort_sync_on_stopped_image` or something more indicative of it's purpose. You can just add one or more commits to vehre/sync_images_fix_new and push them, it will update this PR.

Also can you please add an extra `!` and move these comments below `program ...` that way the documentation will get picked up by [FORD](https://github.com/cmacmackin/ford).